packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts

```diff
@@ -0,0 +1,939 @@
+						placeholder: "",
+						allowTargetBlank: true,
+						single:
+							"heading1,heading2,heading3,heading4,heading5,heading6,strong,em",
+					},
+				},
+				description: {
+					type: "StructuredText",
+					config: {
+						label: "description",
+						placeholder: "",
+						allowTargetBlank: true,
+						multi:
+							"paragraph,preformatted,hyperlink,embed,rtl,strong,em,list-item,o-list-item",
+					},
+				},
+				image: {
+					type: "Image",
+					config: {
+						label: "image",
+						constraint: {},
+						thumbnails: [],
+					},
+				},
+			},
+			items: {
+				title: {
+					type: "StructuredText",
+					config: {
+						label: "title",
+						placeholder: "",
+						allowTargetBlank: true,
+						single: "heading2,heading3,heading4,heading5,heading6,strong,em",
+					},
+				},
+				description: {
+					type: "StructuredText",
+					config: {
+						label: "description",
+						placeholder: "",
+						allowTargetBlank: true,
+						multi:
+							"paragraph,preformatted,hyperlink,strong,em,list-item,o-list-item,rtl",
+					},
+				},
+			},
+		},
+	],
+};
+
+export const screenshotPaths = {
+	default: "AlternateGrid/screenshot-default.png",
+	imageRight: "AlternateGrid/screenshot-imageRight.png",
+};
+
+export const createComponentContents = (
+	model: SharedSlice,
+	isTypeScriptProject: boolean,
+): string => {
+	const pascalName = pascalCase(model.name);
+	if (isTypeScriptProject) {
+		return stripIndent`
+            import { Content, isFilled } from "@prismicio/client";
+            import {
+            SliceComponentProps,
+            PrismicRichText,
+            PrismicImage,
+            } from "@prismicio/react";
+            
+            export type ${pascalName}Props = SliceComponentProps<Content.${pascalName}Slice>;
+            
+            const ${pascalName} = ({ slice }: ${pascalName}Props): JSX.Element => {
+				return (
+					<section
+						data-slice-type={slice.slice_type}
+						data-slice-variation={slice.variation}
+						className="essential-slice es-bounded es-alternate-grid"
+					>
+					<div
+						className={\`
+							es-bounded__content
+							es-alternate-grid__content
+							\${
+								isFilled.image(slice.primary.image)
+								? "es-alternate-grid__content--with-image"
+								: ""
+							}
+						\`}
+					>
+						{isFilled.image(slice.primary.image) && (
+							<PrismicImage
+								field={slice.primary.image}
+								className={\`
+									es-alternate-grid__image
+									\${
+										slice.variation === "imageRight"
+										? "es-alternate-grid__image--right"
+										: "es-alternate-grid__image--left"
+									}
+								\`}
+							/>
+						)}
+						<div className="es-alternate-grid__primary-content">
+							<div className="es-alternate-grid__primary-content__intro">
+								{isFilled.keyText(slice.primary.eyebrowHeadline) && (
+									<p className="es-alternate-grid__primary-content__intro__eyebrow">
+										{slice.primary.eyebrowHeadline}
```

Comment:
No need to use simple PrismicText

Reply:
I don't understand

Reply:
Sorry I wanted to add a question mark haha

Should we use PrismicText for these kind of things?

Reply:
It depends if the designer considers that something else than a p tag should be used. In this case, I reused the first implementation from Sarah and Angelo